ui/src/components/friends/add.rs

```diff
@@ -102,6 +102,17 @@ pub fn AddFriend(cx: Scope) -> Element {
     let username = identity.username();
     let short_name = format!("{}#{}", username, short_id);
     let short_name_context = short_name.clone();
+    let add_friend_lable = if !state.read().ui.is_minimal_view() {
+        get_local_text("uplink.add")
+    } else {
+        String::from("")
```

Comment:
```suggestion
        String::new()
```

Reply:
Never seen this update for incorporating changes, so cool. Great catch, thanks Darius!!